plugins/inputs/interrupts/README.md

```diff
@@ -13,65 +14,99 @@ The interrupts plugin gathers metrics about IRQs from `/proc/interrupts` and `/p
+| - | type | interrupt type |
+| true | cpu | CPU number |
+
+##### interrupts fields 
+| cpu_as_tag | field | counter | units | description |
+|:---:|:---:|:---:|:---:|---|
+| - | count | counter | events | number of times the interrupt has been triggered |
+| false | cpu# | counter | events | number of times the interrupt has been handled by CPU _#_ |
+| false | total | counter | events | total number of times the interrupt has been handled by all CPUs |
+
+#### soft_interrupts
+The `soft_interrupts` measurement reports the soft interrupt data collected from the 
+`/proc/softirqs` file in Linux.
+
+Note: for some Linux systems there can be fixed, large number of CPUs reported in
+the `/proc/softirqs` file. This number could be much larger than the actual number of 
+CPUs in the system. The fields for these phantom CPUs contain zeroes. The approach
+taken to remove these phantom CPUs is to remove the columns containing all zeros
+to the right (higher CPU numbers). For systems where CPUs are dynamically enabled,
+this can lead to CPUs not being reported until enabled. However, this is preferable
+to collecting metrics for tens or hundreds of phantom CPUs. For queries with fixed
+numbers of CPUs, consider using `fill(0)` rather than `fill(null)`
+
+##### soft_interrupts tags
+| cpu_as_tag | tag | description |
+|:---:|:---:|---|
+| - | irq | IRQ name |
+| true | cpu | CPU number |
 
-- interrupts
-  - tags:
-    - irq (IRQ name)
-    - type
-    - device (name of the device that is located at the IRQ)
-    - cpu
-  - fields:
-    - count (int, number of interrupts)
-
-- soft_interrupts
-  - tags:
-    - irq (IRQ name)
-    - type
-    - device (name of the device that is located at the IRQ)
-    - cpu
-  - fields:
-    - count (int, number of interrupts)
+##### soft_interrupts fields 
+| cpu_as_tag | field | type | units | description |
+|:---:|:---:|:---:|:---:|---|
+| false | cpu# | counter | events | number of times the interrupt has been handled by CPU _#_ |
+| false | total | counter | events | total number of times the interrupt has been handled by all CPUs |
+| true | count | counter | events | number of times the interrupt has been handled by CPU in the tag |
+
+
+#### spurious_interrupts
+The `spurious_interrupts` measurement reports the number of spurious interrupts triggered
+and unhandled for IRQs. This data is collected from the `/proc/irq/IRQ/spurious` file.
+This data is identified by IRQ and not per-CPU.
+
+##### spurious_interrupts tags
+| tag | description |
+|:---:|---|
+| device | description of device |
+| irq | IRQ name |
+| type | interrupt type |
+
+##### spurious_interrupts fields 
+| field | type | units | description |
+|:---:|:---:|:---:|---|
+| count | counter | events | number of times the interrupt has been handled (modulo 100,000) |
+| total | counter | events | total number of times the interrupt has been handled |
+| unhandled | counter | events | number of times an interrupt was not handled |
```

Comment:
This looks nice but we need to stick to the previous format style to match https://raw.githubusercontent.com/influxdata/telegraf/master/plugins/inputs/EXAMPLE_README.md

Reply:
will do, though I will assert your standard is ugly and takes up too much screen space :-)